compact/hcal/hcal_backward.xml

```diff
@@ -54,8 +54,11 @@
   <!--  Definition of the readout segmentation/definition  -->
   <readouts>
     <readout name="HcalEndcapNHits">
-      <segmentation type="CartesianGridXY" grid_size_x="100 * mm" grid_size_y ="100 * mm"/>
-      <id>system:8,barrel:3,module:4,layer:8,slice:5,x:32:-16,y:-16</id>
+      <segmentation type="PolarGridRPhi2"
+      grid_r_values="HcalEndcapN_rmin 90.9878*cm 100.89*cm 111.395*cm 122.516*cm 134.229*cm 146.58*cm 159.546*cm 173.155*cm 187.424*cm 202.377*cm 218.019*cm 234.353*cm 251.444*cm HcalBarrel_rmax"
+      grid_phi_values="6*deg 6*deg 6*deg 6*deg 6*deg 6*deg 6*deg 6*deg 6*deg 6*deg 6*deg 6*deg 6*deg 6*deg 6*deg 6*deg 6*deg 6*deg 6*deg 6*deg 6*deg 6*deg 6*deg 6*deg 6*deg 6*deg 6*deg 6*deg 6*deg 6*deg 6*deg 6*deg 6*deg 6*deg 6*deg 6*deg"
+      offset_phi="0.0*deg"/>
+      <id>system:8,barrel:3,module:4,layer:8,slice:5,r:32:-16,phi:-16</id>
```

Comment:
This looks iffy. What determines the grid_r_values? Can we make this clearer?

Reply:
The numbers are based on eta bin limits of STAR EEMC converted from eta to R. There are 12 bins, with 2 extra added (inner and outer ring) to match the dimensions of nHCal. This will need to be updated in the next steps.

Somehow it only managed to compile with all the values in a single line. Didn't work with clearer structure. I'll check if some Windows-style characters/line endings were the culprit.

Reply:
Isn't the segmentation supposed to project along eta? Isn't that lost when you fix the r values? If so, then we should fix GridPhiEta to work in this case.

Reply:
Yes, it would be ideal. So far a pure eta, phi segmentation is missing. I thought that GridPhiEta was supposed to work like that. Looks like it's not the case. I seems GridPhiEta with R=1 is more a feature than a bug. Perhaps we could work with dd4hep developers to add segmentation for pure eta, phi. I don't know the details how segmentation works, so I will need some help.

Reply:
During the calorimeter meeting we discussed that PolarGridRPhi2 is good enough to be ready for the production campaign.

Reply:
Sounds good.